modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -20180,4 +20180,5 @@ zip=Zip
 zip-postal=Zip/Postal
 zoom-in=Zoom In
 zoom-out=Zoom Out
-zoom-to-fit=Zoom to Fit
\ No newline at end of file
+zoom-to-fit=Zoom to Fit
+commerce-order-type-info=The order type cannot be modified beyond this point. If you make an error, please delete the order and begin again.
```

Comment:
If someone is doing a rebase and has conflicts with the build-lang commit, usually the commi is skipped and buildLang is re-run.
This will results in changes that are not coming directly from the user that is doing the rebase (but from your commit) and need to be manually checked. If the base key is already sorted this will not happen.

Reply:
Got it and fixed!
Thanks @luca-pellizzon